services/QuillLMS/spec/controllers/api/v1/activity_sessions_controller_spec.rb

```diff
@@ -47,24 +47,25 @@
         create(:old_concept_result,
           activity_session_id: activity_session.id,
           concept: writing_concept,
-          metadata: { foo: 'bar' }
+          metadata: { foo: 'bar', correct: true }
```

Comment:
Does this `correct: true` have some semantics behind it or is it just another key-value pair?

Reply:
This is a side effect of slightly more strict validation on the new model.  Specifically, SOME value for `correct` is required to build the new-shaped model.  So in this case, the `correct` key is required, but the value could be `true` or `false` (or even truth-y or false-y).